Problem_2.java

```diff
@@ -0,0 +1,73 @@
+/*
+   
+Design Hash-set
+*/
+
+//Time comlpexity - O(log n/k) 

```

Comment:
Which part is making it O(log n/k). As discussed in class time complexity is O(1). Please watch video again